pitest/src/main/java/org/pitest/mutationtest/engine/gregor/config/Mutator.java

```diff
@@ -177,11 +202,72 @@
      */
     add("EXPERIMENTAL_BIG_INTEGER", BigIntegerMutator.INSTANCE);
 
+    /**
+     * Experimental mutators that mutate binary arithmetic operations.
+     */
+    add("EXPERIMENTAL_AOR_1", AOR1Mutator.AOR_1_MUTATOR);
+    add("EXPERIMENTAL_AOR_2", AOR2Mutator.AOR_2_MUTATOR);
+    add("EXPERIMENTAL_AOR_3", AOR3Mutator.AOR_3_MUTATOR);
+    add("EXPERIMENTAL_AOR_4", AOR4Mutator.AOR_4_MUTATOR);
+
+    /**
+     * Experimental mutator that replaces a variable with its negation.
+     */
+    add("EXPERIMENTAL_ABS", ABSMutator.ABS_MUTATOR);
+
+    /**
+     * Experimental mutators that replace a binary arithmetic operations with one of its members.
+     */
+    add("EXPERIMENTAL_AOD1", AOD1Mutator.AOD_1_MUTATOR);
+    add("EXPERIMENTAL_AOD1", AOD2Mutator.AOD_2_MUTATOR);
+
+
+    /**
+     * Experimental mutators that replace an inline constant a with 0, 1, -1, a+1 or a-1 .
+     */
+    add("EXPERIMENTAL_CRCR1", CRCR1Mutator.CRCR_1_MUTATOR);
+    add("EXPERIMENTAL_CRCR2", CRCR2Mutator.CRCR_2_MUTATOR);
+    add("EXPERIMENTAL_CRCR3", CRCR3Mutator.CRCR_3_MUTATOR);
+    add("EXPERIMENTAL_CRCR4", CRCR4Mutator.CRCR_4_MUTATOR);
+    add("EXPERIMENTAL_CRCR5", CRCR5Mutator.CRCR_5_MUTATOR);
+    add("EXPERIMENTAL_CRCR6", CRCR6Mutator.CRCR_6_MUTATOR);
+
+    /**
+     * Experimental mutators that replace an bitwise ands and ors.
+     */
+    add("EXPERIMENTAL_OBBN1", OBBN1Mutator.OBBN_1_MUTATOR);
+    add("EXPERIMENTAL_OBBN2", OBBN2Mutator.OBBN_2_MUTATOR);
+    add("EXPERIMENTAL_OBBN3", OBBN3Mutator.OBBN_3_MUTATOR);
+
+    /**
+     * Experimental mutators that replace conditional operators.
+     */
+    add("EXPERIMENTAL_ROR1", ROR1Mutator.ROR_1_MUTATOR);
+    add("EXPERIMENTAL_ROR2", ROR2Mutator.ROR_2_MUTATOR);
+    add("EXPERIMENTAL_ROR3", ROR3Mutator.ROR_3_MUTATOR);
+    add("EXPERIMENTAL_ROR4", ROR4Mutator.ROR_4_MUTATOR);
+    add("EXPERIMENTAL_ROR3", ROR5Mutator.ROR_5_MUTATOR);
```

Comment:
Should this be `"EXPERIMENTAL_ROR5"`?

```suggestion
    add("EXPERIMENTAL_ROR5", ROR5Mutator.ROR_5_MUTATOR);
```

Reply:
It should, indeed. Thanks!